packages/connectivity/connectivity/pubspec.yaml

```diff
@@ -17,6 +17,8 @@ dependencies:
   flutter:
     sdk: flutter
   meta: "^1.0.5"
+  connectivity_platform_interface:
+    path: ./connectivity_platform_interface
```

Comment:
Please, depend on the published pub ^ version:

`connectivity_platform_interface: ^1.0.2`